cmd/krel/cmd/release_notes.go

```diff
@@ -331,6 +617,65 @@ func tryToFindLatestMinorTag() (string, error) {
 	return strings.TrimSpace(status.Output()), nil
 }
 
+// releaseNotesFor generate the release notes for a specific tag
+func releaseNotesFor(tag string) (*releaseNotesResult, error) {
+	logrus.Infof("Generating release notes for tag %s", tag)
+
+	startTag, err := tryToFindPreviuousTag(tag)
```

Comment:
Generally, what avoids us using the `git.PreviousTag` method?

Reply:
Since the release notes package had another git function `tryToFindLatestMinorTag()` it did not occur to me to look into the git package and created `tryToFindPreviuousTag()`.

Would you rather have me delete this function and use `git.PreviousTag` ?

Reply:
Let's add a TODO and move forward with this PR :+1:

Reply:
OK, done. I've added the todo in a comment.